internal/jimmhttp/rebac_admin/roles.go

```diff
@@ -0,0 +1,230 @@
+// Copyright 2024 Canonical.
+
+package rebac_admin
+
+import (
+	"context"
+
+	v1 "github.com/canonical/rebac-admin-ui-handlers/v1"
+	"github.com/canonical/rebac-admin-ui-handlers/v1/resources"
+	"github.com/juju/names/v5"
+
+	"github.com/canonical/jimm/v3/internal/common/pagination"
+	"github.com/canonical/jimm/v3/internal/errors"
+	"github.com/canonical/jimm/v3/internal/jimmhttp/rebac_admin/utils"
+	"github.com/canonical/jimm/v3/internal/jujuapi"
+	ofganames "github.com/canonical/jimm/v3/internal/openfga/names"
+	apiparams "github.com/canonical/jimm/v3/pkg/api/params"
+	jimmnames "github.com/canonical/jimm/v3/pkg/names"
+)
+
+// rolesService implements the `RolesService` interface.
+type rolesService struct {
+	jimm jujuapi.JIMM
```

Comment:
hmm.. should we instead require 
```
type RoleManager interface {
 ...
}

type RoleManagerGetter interface {
  GetRoleManager() RoleManager
}
```
then this would become 
```
type rolesService struct {
  jimm RoleManagerGetter
}
```

naming, obviously, is hard

Reply:
I'd prefer to keep it as is for consistency and then switch them over once we no longer have the `RoleManagerGetter`.